pkgs/servers/monitoring/nagios-plugins/manubulon-snmp-plugins/default.nix

```
@@ -0,0 +1,197 @@
+{
+  fetchDebianPatch,
+  fetchFromGitHub,
+  lib,
+  makeWrapper,
+  manubulon-snmp-plugins,
+  nix-update-script,
+  perlPackages,
+  stdenv,
+  testers,
+}:
+stdenv.mkDerivation rec {
+  pname = "manubulon-snmp-plugins";
+  version = "2.1.0";
+
+  src = fetchFromGitHub {
+    owner = "SteScho";
+    repo = "manubulon-snmp";
+    rev = "v${version}";
+    sha256 = "sha256-I4wKAy704ddoseb7NFBj7+O7l1Sy2JItoe6wrBUaJRQ=";
```

```suggestion
    hash = "sha256-I4wKAy704ddoseb7NFBj7+O7l1Sy2JItoe6wrBUaJRQ=";
```

`sha256` attribute is deprecated. `hash` is a replacement.

---

Done.